reactive/load_balancer.py

```diff
@@ -168,7 +162,21 @@ def install_load_balancer():
 
         port = hookenv.config('port')
         hookenv.open_port(port)
-        services = apiserver.services()
+        services = []
+        if apiserver:
+            services.extend(apiserver.services())
+        for request in lb_consumers.all_requests:
```

Comment:
We should probably verify the request protocol is http/https, since that's all we can really loadbalance to.